### PR TITLE
Make this repo compatible with DAPS & packaging as RPM 

### PR DESCRIPTION
**Pull request by ghost**
This branch introduces the following:

* moves adoc files into "adoc" dir as required by DAPS, and adds a DC (doc config) file, also required by DAPS
* introduces `{textMacros}` for the product name etc. Unfortunately that hurts the display in GitHub a little but it protects you from typos in product names etc.
* adds a Makefile, so we can later build this as an RPM package (see https://build.opensuse.org/package/show/Documentation:Auto:ASCIIDoc/release-notes-cap)
* cleans up markup a bit, partly just because, partly to look better in DAPS
* adds a LICENSE -- I chose CC-BY which I think is a sensible choice, but please tell me if it should be something else instead

Also, note that this PR bitrots with each new merge to master which makes updating this a bit painful.
To ease this, a3c6ef0 and a1f7b8e are currently separate, but they should be squashed before merging, so Git will recognize that the files have actually been moved.

To finalize the publication of this, the following four items still need to be done after merging:

* The OBS package needs to be set up to grab content from `master` instead of `daps-compat`.
* The `Obs` integration for GitHub needs to be activated (so changes on `master` automatically trigger a build on OBS) 
* The internal config file for release notes needs to be adapted
* The internal config file for which release notes to display on suse.com/releasenotes needs to be updated

### CODE REVIEW

File: DC-release-notes [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## --------------------------------- 
+## Doc Config File for openSUSE Leap
```
Comment (cornelius) [inline DC-release-notes]: This should be "SUSE Cloud Application Platform", not openSUSE Leap, I suppose.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	$(DAPS_COMMAND) -d $< html --single \
+	  --param="toc.section.depth=2" \
+	  $(webaccessparams) \
+	  $(buglink)
+
+
+yast-html: build/release-notes/yast-html/release-notes.html
+build/release-notes/yast-html/release-notes.html: $(YAST_PROFILED_FILE)
+	mkdir -p build/release-notes/yast-html/; \
+	$(XSLTPROC_COMMAND) $(daps_xslt_rn_dir)/yast.xsl $< > $@
```
Comment (cornelius) [inline Makefile]: Do we need that? CAP can't be installed with YaST, so we would never show its release notes in YaST.

File: adoc/MAIN.release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+:rn-url: https://www.suse.com/releasenotes
+:doc-url: https://www.suse.com/documentation/cloud-application-platform-1
+:deployment-url: https://www.suse.com/documentation/cloud-application-platform-1/book_cap_deployment/data/book_cap_deployment.html
+:suse: SUSE
+:caasp: {suse} Containers as a Service Platform
+:caaspa: {suse} CaaS Platform
+:ostack: OpenStack
+:cf: Cloud Foundry
+:scc: {suse} Customer Center
+:azure: Microsoft Azure
```
Comment (cornelius) [inline adoc/MAIN.release-notes.adoc]: As far as I can tell these variables are not expanded on GitHub. So the texts only can reasonably be read after the rendering step through DAPS, right?

Is there a way how the texts could be rendered in a human-readable way on GitHub already? That's what we have right now and it is useful to get early feedback and prepare notes before they end up in the official release notes or in the documentation.

**Review by cornelius — COMMENTED** [review]
The changes look good in general. Thanks for reviewing the texts. I added a few comments on specific parts of the changes.

**Comment by cornelius** [comment]
I checked with the team. License is ok.

Regarding the human-readable version on GitHub, I would be ok with merging it as it is for now if there is no easy solution.

The Azure document will eventually go into the official manual. There already is a draft so we don't need to spend too much time on Notes-Azure.adoc here.

**Comment by ghost** [comment]
@cornelius This should fix all the issues you had -- essentially GitHub preview should work now. The attribute entries (i.e. those `:entry: some value`) are now all stored centrally in a file `entities.adoc`. For DAPS building, this file is simply included.
GitHub does not support includes, therefore there is now a process to update the attribute entries in each of the adoc files -- `make update-entries`.

I have also rebased on top of current develop and manually added the new changes to my new files.

**Review by cornelius — APPROVED** [review]
Looks great now.

**Pull request merged**